bot/__main__.py

```diff
@@ -1,8 +1,9 @@
+import asyncio
 import inspect
 import pkgutil
 from typing import Iterator, NoReturn
 
-from disnake import AllowedMentions, Intents
+from discord import AllowedMentions, Intents
 
 from . import settings
```

Comment:
Absolute imports are better

Reply:
We could change this in the future, currently relative imports are used throughout the project wherever possible.

Reply:
bet